filter/etc/e-smith/templates/etc/rspamd/rspamd.local.lua/00template_vars

```diff
@@ -0,0 +1,6 @@
+{
+  @symbols = qw(
+            SPOOF_REPLYTO
+  );
+'';
+}
```

Comment:
Remember to remove the EOL at EOF!

Reply:
?????

Reply:
This fragment should produce no output at all. Here the file ends with a "\n", so it expands to "\n". 

You ought remove it.

Reply:
this fragment output a `1` if I do not  use ` '';`

How to solve it ?

Reply:
It's a very little thing I often ask you to do, but you can't. I'll do it for you :kissing:

Reply:
mainly I did not understand I think, I am sorry